### PR TITLE
Adding Token and UIToken variables to make "shortcuts" for TokenProvider and UITokenProvider

### PR DESCRIPTION
**Pull request by dinsen**
I think we talked about it before @mohsen-schibsted. That it would be great if developers only needed to type `Warp.Token.text` instead of `Warp.Config.tokenProvider.text`.

This PR is a suggestion to make new public variables that the developers should use, instead of using the variables from `Config`.

Later we should remove the public keyword from the variables in `Config`. 
Since this is a breaking change, I haven't removed the public keyword. 

Let's have a discussion if we should call it something different? 
And what about colors. Should they also be public available?

### CODE REVIEW

**Comment by mohsen-schibsted** [comment]
> I think we talked about it before @mohsen-schibsted. That it would be great if developers only needed to type `Warp.Token.text` instead of `Warp.Config.tokenProvider.text`.
> 
> This PR is a suggestion to make new public variables that the developers should use, instead of using the variables from `Config`.
> 
> Later we should remove the public keyword from the variables in `Config`. Since this is a breaking change, I haven't removed the public keyword.
> 
> Let's have a discussion if we should call it something different? And what about colors. Should they also be public available?

Thank you, this looks great 🥳 Generally no people should not have access to Colors and should use Tokens. Colors are for internal use in WARP components. But since we have cases that we had to sacrifice and use WARP colors in some places in some edge cases we still may need to make some of them public.
So yes please make one for Colors and UIColors also, and maybe deprecate the ones inside Config too so people stop using that and we can remove them later on.

File: Sources/Config.swift [hunk 1]
```diff
@@ -1,4 +1,12 @@
 extension Warp {
+    public static var Token: TokenProvider {
+        return Config.tokenProvider
```
Comment (mohsen-schibsted) [inline Sources/Config.swift:3]: Maybe just copy and paste that code to here and just deprecate the ones inside Config so other people stop using those too 👍

**Review by mohsen-schibsted — APPROVED** [review]

File: Sources/Config.swift (hunk 1 above)
Comment (dinsen) [reply Sources/Config.swift:3]: My plan was to get this into a new release.
Fix the NMP project and then deprecate/remove the old variables in a new Warp release after.
Just to ensure the project always can compile. But again. we have a summer release freeze in NMP, so maybe we should just make the cut? 🤔

Comment (mohsen-schibsted) [reply Sources/Config.swift:3]: I say just go for it 😅 should be fine. It is just a deprecation so only warnings even if someone is using it.

Comment (mohsen-schibsted) [reply Sources/Config.swift]: Looks great 🥳

**Pull request merged**